src/seedu/addressbook/AddressBook.java

```diff
@@ -208,7 +214,20 @@
 
     public static void main(String[] args) {
         showWelcomeMessage();
-        processProgramArgs(args);
+        
+        if (args.length > MAX_NUMBER_OF_ARGUMENTS) {
```

Comment:
good that magic constants are replaced. but why take the code out of `processProgramArgs`?